WebRoot/skins/_base/base3/splash.html

```diff
@@ -20,42 +20,91 @@
  * All portions of the code are Copyright (C) 2011, 2012, 2013, 2014, 2016 Synacor, Inc. All Rights Reserved.
  * ***** END LICENSE BLOCK *****
 -->
-<!-- BEGIN SPLASH SCREEN -->
-<div id='skin_container_splash_screen' class='SplashScreen'>
-	<script language='javascript'>
-		function showCompanyUrl() {
-			window.open(ZmMsg.splashScreenCompanyURL, '_blank');
-		}
-	</script>
-	
-	<div id="ZSplashPanel" class="center">
-		<div class="contentBox">
-			<h1><div id='ZLoginBannerImage' class='ImgLoginBanner' onclick='showCompanyUrl()'></div></h1>
-			<div id="ZLoginAppName">
-				<script>if(!window.isOffline) { document.write(ZmMsg.splashScreenAppName) }</script>
-			</div>
-			<div id="#ZSplashBodyContainer" class="content">
-				<div class="message">
-					<script>document.write(ZmMsg.splashScreenLoading)</script>
-				</div>
-				<div class="switch" id="splashScreenSwitchContainer">
-					<script>if(!window.isOffline) { document.write(ZmMsg.splashScreenSwitch) }</script>
-				</div>
-			</div>
-		</div>
-        <div class="decor1"></div>
-	</div>
-	<div class="Footer">
-        <div class="copyright">
-          <script>
-            if (window.isOffline) {
-              document.write(ZmMsg.splashScreenCopyrightZD);
-            } else {
-              document.write(ZmMsg.splashScreenCopyright);
-            }
-          </script>
-        </div>
-    </div>
-	<div class="decor2"></div>
-</div>
-<!-- END SPLASH SCREEN -->
+<!-- BEGIN SPLASH SCREEN -->
```

Comment:
When I see code diff on github, most of the code is treated as 'changed'. Has the line break type (LF, CR, CR+LF) been changed? It should not be changed typically.

Reply:
Fixed